### PR TITLE
Hotel page update

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review
* changes the style of the hotel page  
#487

### CODE REVIEW

File: src/components/hotel/index.js [hunk 1]
```diff
@@ -107,20 +108,21 @@ class HotelPage extends Component {
...
           <div className='factHeading'>Hotelfakta</div>
           <div className='factContainer'>
-            <div className='factTitles'>
-              <div>Strand</div>
-              <div>Lokalt centrum</div>
-              <div>Pool</div>
-              {this.renderAmenities(amenities)}
+            <div className='factPair'>
+              <div className='fact'>Strand</div>
+              <div className='value'>{this.renderFact('distancetobeach')}m</div>
```
Comment (Kumjami) [inline src/components/hotel/index.js]: Is it working?

Comment (jackcarlisle) [inline src/components/hotel/index.js]: will fix this morning

File: src/components/hotel/index.js [hunk 2]
```diff
@@ -109,10 +114,6 @@ class HotelPage extends Component {
           <div className='factHeading'>Hotelfakta</div>
           <div className='factContainer'>
             <div className='factPair'>
-              <div className='fact'>Strand</div>
-              <div className='value'>{this.renderFact('distancetobeach')}m</div>
-            </div>
-            <div className='factPair'>
               <div className='fact'>Lokalt centrum</div>
               <div className='value'>{this.renderFact('distancetocenter')}m</div>
```
Comment (tomgco) [inline src/components/hotel/index.js]: We should also do this for distance to centre as well?

**Comment by Kumjami** [comment]
looks good :ok_hand:

**Pull request merged**